com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -61,6 +61,26 @@ public enum ScriptableRenderPassInput
         /// </summary>
         AfterRenderingPrePasses = 200,
 
+        /// <summary>
+        /// Executes a <c>ScriptableRenderPass</c> before rendering gbuffer pass.
+        /// </summary>
+        BeforeRenderingGbuffer = 210,
```

Comment:
What happens if a render pass is schedule in BeforeRenderingGBuffer but executed in Forward mode?
Do we execute or skip it?

Reply:
I think we would execute it.
Gbuffer pass does not contain logic to clear the camera color attachment, it is implicitly done in the infamous `ScriptableRenderer.SetRenderPassAttachments()` method. So I expect the render pass to execute correctly, then gbuffer pass and deferred ligthing pass write on top of it without clearing camera color attachment.
Normally forward passes should be added with `BeforeRenderingOpaques`/`AfterRenderingOpaques`.